tools/mason/MasonNew.chpl

```diff
@@ -112,9 +112,13 @@ considered if no input is given.""");
   var packageName: string;
   var defPackageName: string = defaultPackageName;
   var version: string;
-  var defaultVersion: string = "0.1.0";
+  var defaultVersion: string;
+  if defVer == '' then defaultVersion = "0.1.0";
```

Comment:
Note you can also write this as:

```chpl
var defaultVersion  = if defVer == ''" then "0.1.0" else defVer;
```

Reply:
Didn't know we could do that, thanks @ben-albrecht